buildSrc/src/main/groovy/airbyte-integration-test-java.gradle

```diff
@@ -0,0 +1,42 @@
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.tasks.testing.Test
+
+class AirbyteIntegrationTestJavaPlugin implements Plugin<Project> {
+    void apply(Project project) {
+        project.sourceSets {
+            integrationTest {
+                java {
+                    srcDir 'src/test-integration/java'
+                }
+                resources {
+                    srcDir 'src/test-integration/resources'
+                }
+            }
+        }
+        project.test.dependsOn('compileIntegrationTestJava')
+
+        project.configurations {
+            integrationTestImplementation.extendsFrom testImplementation
+            integrationTestRuntimeOnly.extendsFrom testRuntimeOnly
+        }
+
+        project.task('integrationTest', type: Test) {
+            testClassesDirs += project.sourceSets.integrationTest.output.classesDirs
+            classpath += project.sourceSets.integrationTest.runtimeClasspath
+
+            useJUnitPlatform()
+            testLogging() {
+                events "passed", "failed"
+                exceptionFormat "full"
+            }
+
+            if(project.hasProperty("airbyteDocker")) {
```

Comment:
shouldn't this dependOn belong in the airbyte-docker?

Reply:
I don't think so. I actually just did something like this for the source test too, since `airbyte-docker` should be applied first, before `airbyte-integration-test-java` exists.